src/components/header/Header.jsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import "../../style/header.scss";
+import AppRouting from "../../AppRouting";
+
+const routes = [
```

Comment:
I would put this in constants or better yet in a config file. Where there were two objects, one contained the route constants. The second contained router data. Below is an example. I would also store everything in the routes folder
**routeConfig.js**
```
export const ROUTES_LIST = [
  {
          id: 1,
          name: 'home page',
          value: 'Home Page',
          path: '/',
  },
  {
          id: 2,
          name:  'examples',
          value: 'Examples',
          path: '/examples',
  }
   .....
```
```
export const routes = [
    {
        path:  ROUTES_LIST.path,
        component: HomePage,
        exact: true
    },
    {
        path: ROUTES_LIST.examples,
        component: Examples,
        exact: true
    },
    ....
]

```